portal-bridge/src/beacon_bridge.rs

```diff
@@ -298,21 +311,41 @@ impl BeaconBridge {
     async fn serve_light_client_finality_update(
         api: ConsensusApi,
         portal_clients: Arc<Vec<HttpClient>>,
-    ) -> anyhow::Result<()> {
+        finalized_slot: u64,
+    ) -> anyhow::Result<u64> {
         let data = api.get_lc_finality_update().await?;
         let update: Value = serde_json::from_str(&data)?;
         let update: LightClientFinalityUpdateCapella =
             serde_json::from_value(update["data"].clone())?;
         info!(
-            "Got lc finality update for slot {:?}",
-            update.attested_header.beacon.slot
+            finalized_slot = %update.finalized_header.beacon.slot,
+            "Got LightClientFinalityUpdate",
```

Comment:
If I read this log without context, I would have thought it meant: "Received LightClientFinalityUpdate from peer" rather than "Generated LightClientFinalityUpdate for bridging broadcast". Maybe `Generated LightClientFinalityUpdate`?